frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-overview-details-section-card/feature-flag-overview-details-section-card.component.ts

```diff
@@ -31,22 +31,32 @@ import { Subscription } from 'rxjs';
   styleUrl: './feature-flag-overview-details-section-card.component.scss',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class FeatureFlagOverviewDetailsSectionCardComponent {
+export class FeatureFlagOverviewDetailsSectionCardComponent implements OnInit, OnDestroy {
   isSectionCardExpanded = true;
   @Output() sectionCardExpandChange = new EventEmitter<boolean>();
+  featureFlag: FeatureFlag;
```

Comment:
the `flag`, `flag sub`, we have `$featureFlag` already and `flag` is usable in the template, just pass it into the menu item click callback function to get the flagId to pass.

the `emailId` can stay, I wish we had a better way of getting that value because I really want to avoiding subbing to the store like this just to get values "out" of observables, because all local state that isn't observable becomes a liability of the component.

But... alternatives to just subbing here don't look great either, there's not a good logical place in the template for an `emailId$ | async` kind of thing. So I guess let's leave the `emailId` part of this, I don't have a better idea right now.